### PR TITLE
Adding final non-HappyBase Bigtable system tests.

### PR DESCRIPTION
**Pull request by dhermes**

### CODE REVIEW

**Comment by dhermes** [comment]
@jonparrott FYI I ran this on top of #1532, but assume it would work on the older set-up as well.

File: system_tests/bigtable.py [hunk 1]
```diff
@@ -384,3 +390,95 @@ def test_read_row(self):
...
+        cell1, cell2, cell3, cell4 = self._write_to_row(row, row_alt,
+                                                        row, row_alt)
+        row.commit()
+        row_alt.commit()
+
+        rows_data = self._table.read_rows()
+        self.assertEqual(rows_data.rows, {})
+        rows_data.consume_all()
+
+        # NOTE: We should refrain from editing protected data on instances.
```
Comment (jonparrott) [inline system_tests/bigtable.py]: What's stopping us from resolving this now? Should we file an issue instead?

**Comment by jonparrott** [comment]
LGTM with small concern.

File: system_tests/bigtable.py (hunk 1 above)
Comment (dhermes) [inline system_tests/bigtable.py]: The joys of copying and pasting code (even though I wrote it). I'll file an issue but agree that it should be tackled. I'm kind of at a loss for how to tackle this, other than providing a factory. WDYT?

Comment (jonparrott) [inline system_tests/bigtable.py]: `FactoryFactoryImpl`.

But in all seriousness I don't have very strong feelings about tests messing with private attributes. It's up to you.

Comment (dhermes) [inline system_tests/bigtable.py]: FIled #1535

**Pull request merged**